session.go

```diff
@@ -299,6 +299,8 @@ func (s *Session) Subscribe(
 		}
 		return nil, err
 	}
+	rt.subscribeID = id
+	rt.trackAlias = alias
```

Comment:
Should we add an `init` function to make sure users always have to pass `subscribeID`, `trackAlias`, and the `onUnsubscribe` callback?

Reply:
Yes, that is probably a good idea since we always send these fields.